src/inc/corinfo.h

```diff
@@ -1813,6 +1813,9 @@ struct CORINFO_EE_INFO
         unsigned    offsetOfCalleeSavedFP;
         unsigned    offsetOfCallTarget;
         unsigned    offsetOfReturnAddress;
+#ifdef _TARGET_ARM_
```

Comment:
We try to avoid ifdefs in the JIT/EE interface. It causes problems in the cross-compilation scenarios. 

Could you please change this field to be in everywhere, and just add comment that it is applicable for ARM only?

Reply:
Ok, makes sense.